Applications/Appium-TLP.md

```diff
@@ -0,0 +1,182 @@
+* Many tools have been created and maintained by the community, for example a desktop app [GUI](https://github.com/appium/appium-dot-app) wrapper for the server, or a [REPL](https://github.com/Jonahss/AppiumRepl).
+* Many highly-viewed tutorials have been created by the community, even on commercial e-learning sites (like [this one on Udemy](https://www.udemy.com/appium-selendroid-tutorials/)).
+* Dozens of conference talks have been given at conferences and meetups around the world, for example at [LXJS 2013](https://www.youtube.com/watch?v=zsbNVkayYRQ).
+* Appium has become an integral part of many testers' careers. For example, [indeed.com](http://www.indeed.co.uk/jobs?q=appium&l=London) [searches](http://www.indeed.com/jobs?limit=10&as_not=&sort=&jt=all&as_any=&l=San+Francisco%2C+CA&fromage=any&as_cmp=&psf=advsrch&as_phr=&as_ttl=&st=&salary=&as_and=appium&radius=50) for SF, CA and London, UK each show > 100 open jobs with "appium" in the job requirements.
+
+### Contribution Process
+
+Our
+[CONTRIBUTING.md](https://github.com/appium/appium/blob/master/CONTRIBUTING.md)
+formalizes the various kinds of contributions and the processes necessary for
+each. More contribution resources are available:
+
+* [Style Guide](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/style-guide-2.0.md)
+* [Developer's Overview](https://github.com/appium/appium/blob/1.5/docs/en/contributing-to-appium/developers-overview.md)
+* [Run-down of Appium packages and organization](https://github.com/appium/appium/blob/1.5/docs/en/contributing-to-appium/appium-packages.md)
+* [How to run Appium from source](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/appium-from-source.md)
+
+
+### Governance Process
+
+* Our project's roles and procedures are documented on our [GOVERNANCE.md](https://github.com/appium/appium/blob/1.5/GOVERNANCE.md).
+
+### Organizational Tools
+
+* Website: [appium.io](http://appium.io)
+* [Docs](http://appium.io/slate/en/master) (Jekyll + Slate)
+* Source code management: [GitHub](https://github.com/appium/appium)
+	* Appium is split up into dozens of smaller repos and operates as an independent GitHub organization
+* Issue tracking: [GitHub](https://github.com/appium/appium/issues)
+* Milestone tracking: [GitHub](https://github.com/appium/appium/milestones)
+* Releases: [GitHub](https://github.com/appium/appium/releases) and [NPM](https://npmjs.org/package/appium)
+* User discussion forums: [discuss.appium.io](https://discuss.appium.io)
+* User chat: [gitter.im/appium](https://gitter.im/orgs/appium/rooms) (#appium on Freenode is mostly abandoned)
+* Developer/contributor chat: [appium.slack.com](https://appium.slack.com)
+* Developer mailing list: [appium-developers@googlegroups.com](https://groups.google.com/forum/#!forum/appium-developers)
+
+### Intellectual Property and Other Assets
+
+Appium does not currently have any registered trademarks. We own/manage the
```

Comment:
Just note the assets and who currently owns them so that at some point in the future the TSC could hand that off to the foundation lawyers to go and have transferred.